Wbooks/src/app/screens/navigator.js

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
```

Comment:
For my grandma's underwear!!! You are importing all react here, i think is not necessary. Delete this and try the app, it should work.

Reply:
![mlb-reaction-pablo-sandoval-1ipgzFqnollIuA4OdL](https://media2.giphy.com/media/1ipgzFqnollIuA4OdL/giphy.gif)